Library/Homebrew/cmd/config.rb

```diff
@@ -119,8 +119,10 @@ def describe_system_ruby
     s
   end
 
-  def java_version
-    java = `java -version 2>&1`.lines.first.chomp
+  def describe_java
+    java = which "java"
+    return "N/A" if java.nil?
+    java = `#{java} -version 2>&1`.lines.first.chomp
```

Comment:
So what's the `/usr/libexec/java_home` output on a clean system? Is the the same as `/usr/libexec/java_home --exec java -version`?